projects/addon-doc/src/components/code/code.template.html

```diff
@@ -1,4 +1,7 @@
 <p *ngIf="filename" class="t-header">{{filename}}</p>
-<pre
-    class="t-code"
-><code [lineNumbers]="true" [highlight]="processor$ | async"></code></pre>
+<ng-container *ngFor="let block of processor$ | async; let count = count">
+    <pre
+        class="t-code"
+    ><code [lineNumbers]="true" [highlight]="block.content"></code></pre>
+    <br *ngIf="count > 1" />
```

Comment:
Why do we need a `br`?

Reply:
![image](https://user-images.githubusercontent.com/12021443/149722259-12da1ebf-5a74-4deb-999b-a3894c5c3635.png)

If you put several blocks of code in a file, then they should have a break. Otherwise, if you remove the `br` tag

![image](https://user-images.githubusercontent.com/12021443/149722427-1b778b4f-1b04-492b-9a18-84f332cc5c6a.png)

Reply:
Got it. I think we can handle this with CSS:
```css
.t-code + .t-code {
  margin-top: 1rem;
}
```

Reply:
Great idea, every time forgot about css, thank you! Fixed